src/app/[locale]/_icons/firework.tsx

```diff
@@ -2,8 +2,8 @@ const Firework = () => (
   <svg width="60" height="60" viewBox="0 0 60 60" fill="none" xmlns="http://www.w3.org/2000/svg">
     <mask id="mask0_41_17491" maskUnits="userSpaceOnUse" x="0" y="0" width="60" height="60">
       <path
-        fill-rule="evenodd"
-        clip-rule="evenodd"
+        fillRule="evenodd"
+        clipRule="evenodd"
```

Comment:
Why this changes...? 🤔

Reply:
fill-rule and clip-rule were triggering warnings on the browser console that way (when images are loaded in profile page)